### PR TITLE
Base configs+

### PR DESCRIPTION
**Pull request by madragonse**
Bese configs separated to _{feature} directories. Provided examplary configs arount checkpointing and fine-tuning llama 3 1B. Fine-tuning of llama works, even with not matched tokenizer. Checkpointing and conversion of HF model to Nano works.

Fine tuning vs pretrainig of Llama 3 1B (mismatched tokenizer):
image

Exps with Llama 3 1B (from oldest: fine tuning of HF llama3 1B, pretraining of llama3 1B, save HF to Nano, fine-tune Nano):
https://app.neptune.ai/o/pmtest/org/llm-random/runs/compare?viewId=9f83cbd8-5a04-4ab7-b9ee-fe78ac58199d&dash=charts&query=((%60sys%2Ftags%60%3AstringSet%20CONTAINS%20%22nano%22)%20AND%20(%60sys%2Ftags%60%3AstringSet%20CONTAINS%20%22pc%22))&sortBy=%5B%22sys%2Fcreation_time%22%5D&sortFieldType=%5B%22datetime%22%5D&sortFieldAggregationMode=%5B%22auto%22%5D&sortDirection=%5B%22descending%22%5D&suggestionsEnabled=false&lbViewUnpacked=true&compareChartsFilter=loss&compare=MwNgnADA7ANAjHeEYCYg

### CODE REVIEW

**Comment by madragonse** [comment]
HALTED- HF -> Nano -> FT is diefferent than HF -> FT:

image

**Comment by madragonse** [comment]
@crewtool you have said that this is ok, so after commented changes PR is ready to merge?

File: configs/fine_tune_llama_hf.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+trainer:
+  n_steps: 2000
+  learning_rate: 0.00005
+  gradient_accumulation_steps: 4
+
+infrastructure:
+  metric_logger:
+    name: test
+    project_name: pmtest/llm-random
```
Comment (madragonse) [inline configs/fine_tune_llama_hf.yaml:21]: move to misc

File: configs/fine_tune.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  - _model@_here_: llama_3_1
+  - _trainer@_here_: llama
+  - _dataset@_here_: c4
+  - _checkpoints@_here_: none
+  - _misc@_here_: default
+
+infrastructure:
+  metric_logger:
+    name: test
+    project_name: pmtest/llm-random
```
Comment (madragonse) [inline configs/fine_tune.yaml:14]: move to misc

File: configs/_checkpoints/none.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+trainer:
+  checkpoint:
+   save:
+    interval: -1
+    path: null
+    model_checkpoint_filename: null
```
[the remark this answers is not in the document]
Comment (madragonse) [reply configs/_checkpoints/none.yaml:6]: done

File: configs/save_llama_hf.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# _run_: true
```
Comment (crewtool) [inline configs/save_llama_hf.yaml:1]: remvoe
